### DISCUSSION TITLE
"Bus error (core dumped)" in diamond blastx

### DISCUSSION THREAD

**Discussion by tamdao216** [bbuchfink/diamond#778] [Q&A]
I am following this protocol "DIAMOND+MEGAN: Fast and Easy Taxonomic and Functional Analysis of Short and Long Microbiome Sequences" and stuck at the aligment step. the details is in attached photo. Please help me if you know this error.
~/users/tam/tools/diamond blastx -d ~/users/tam/diamond/nr -q ~/users/tam/diamond/OCM_join/S2_merged.fastq -o ~/users/tam/diamond/blastx/S2_merged.daa -f 100
![KakaoTalk_20240111_230549929](https://github.com/bbuchfink/diamond/assets/119151272/c3f70e36-4e1e-452d-a332-d58005d590a3)

**Comment by bbuchfink** [comment]
What diamond version are you using and what's your system configuration? Does the same error occur on a different system?

**Comment by tamdao216** [comment]
hi, 
I used diamond 2.1.8 version.
I run Diamond on a server with 48 CPUs and 252GB of RAM.
it is fine when I run a test for few Mb input file, but when I change to like 12 Gb input file, it always crash randomly.

**Reply by bbuchfink** [reply]
Is it a linux system? Intel/AMD or something else? Can you try using the latest version?

**Reply by tamdao216** [reply]
this one Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz.
Yes I think 2.1.8 is the lastest version.
Is there any way I can run small files and then merge them into 1 file?

**Reply by bbuchfink** [reply]
I don't know about the error. You can make your input file available to me and I can try to reproduce it.

It is possible to merge the results for smaller files, using `diamond merge-daa`.

**Reply by tamdao216** [reply]
thank you for your support. I will email you my input. I tried to split my in put to small files but the error still occurs. I will describe detail in the email.
